components/eamxx/src/physics/shoc/tests/CMakeLists.txt

```diff
@@ -75,27 +73,36 @@ set(SHOC_TESTS_SRCS
 
 # NOTE: tests inside this if statement won't be built in a baselines-only build
 if (NOT SCREAM_BASELINES_ONLY)
-  CreateUnitTest(shoc_tests    "${SHOC_TESTS_SRCS}" "${NEED_LIBS}"    THREADS 1 ${SCREAM_TEST_MAX_THREADS} ${SCREAM_TEST_THREAD_INC} DEP shoc_tests_ut_np1_omp1)
+  CreateUnitTest(shoc_tests    "${SHOC_TESTS_SRCS}"
+    LIBS shoc
+    THREADS 1 ${SCREAM_TEST_MAX_THREADS} ${SCREAM_TEST_THREAD_INC}
+  )
+
   if (NOT SCREAM_SMALL_KERNELS)
-    CreateUnitTest(shoc_sk_tests "${SHOC_TESTS_SRCS}" "${SK_NEED_LIBS}" THREADS 1 ${SCREAM_TEST_MAX_THREADS} ${SCREAM_TEST_THREAD_INC} DEP shoc_tests_ut_np1_omp1 EXE_ARGS shoc_main_bfb)
```

Comment:
After discussing with @tcclevenger , I'm pretty sure all usages of `DEP` were caused by copy+paste previous code. I suspect this was originally needed (eons ago) when p3 used to compute some tables on the fly. In that case, we had to ensure only one test wrote to file. A while ago we moved the tables to the inputdata folder, so we no longer compute them. The p3 tests are indeed free of the DEP keyword, but I suspect it got copy+pasted in shoc, and from there in tms and doubly_periodic tests. If you can think of a reason why we still need DEP, let me know, and I'll revert the commit.